src/i18n/de.json

```
@@ -643,6 +643,7 @@
         "bank-transfer-amount": "Betrag",
         "bank-transfer-qrcodes": "QR-Codes für die Überweisung",
         "amount": "Dir werden <em>einmalig</em> {{amount}}&nbsp;€ abgebucht.",
+        "amount-after-fees": "Wir kriegen davon {{amount}}&nbsp;€",
```

I would like to make it more clear that this is due to fees as that may not be apparent to everyone. Maybe like this?

```suggestion
        "amount-after-fees": "Nach Gebühren kriegen wir {{amount}}&nbsp;€.",
```